crates/aide/src/axum/inputs.rs

```diff
@@ -163,7 +163,9 @@ where
     T: JsonSchema,
 {
     fn operation_input(ctx: &mut crate::generate::GenContext, operation: &mut Operation) {
-        let schema = ctx.schema.subschema_for::<T>().into_object();
+        // `subschema_for` `description` is none, while `root_schema_for` is some
+        let schema = ctx.schema.root_schema_for::<T>().schema;
+        operation.description = schema.metadata.as_ref().and_then(|x| x.description.clone());
```

Comment:
I'm pretty skeptical about this change. Can you split it into its own PR?

Reply:
i have 3 options:

- change description for Path (macro and manual can use Path extract). IN THIS PR NOW.

- generate impl for OperationInput as part of `axum_typed_path` and use `root_schema_for` (attribute macro works as before; aide with extract tag does not work anymore)

- make extract TypedPath by analogy with Path, so manual can tell aide macro proper extractor

Reply:
Adding a `TypedPath` "extractor" seems like a good idea. Can you name it `__TypedPath` and make it `#[doc(hidden)]`? Since users never need to use it directly; then we don't need to implement `FromRequestParts` for it either since it's only going to be used for `input_with`.

Reply:
Well.. I guess it's fine to make it public.